gen/tinia

```diff
@@ -47,21 +48,28 @@ foreach my $resourceId ( $data->getResourceIds() ) {
 	my $resourceArming = $data->getResourceAttributeValue( resource => $resourceId, attrName => $A_RESOURCE_ARMING );
 	my $resourceAccGroup = $data->getResourceAttributeValue( resource => $resourceId, attrName => $A_RESOURCE_ACC_GROUP );
 
+	#skip resource with COVID-APPROVAL (for now)
+	if( $resourceAccGroup eq 'COVID-APPROVAL' ) { next; }
```

Comment:
is there a reason to have these conditions here? (instead of unassigning the resources from the service)?